appeals/web/src/server/lib/mappers/appeal/appeal.mapper.js

```diff
@@ -0,0 +1,174 @@
+import config from '#environment/config.js';
+import { generateIssueDecisionUrl } from '#appeals/appeal-details/issue-decision/issue-decision.mapper.js';
+import {
+	mapDocumentDownloadUrl,
+	mapVirusCheckStatus
+} from '#appeals/appeal-documents/appeal-documents.mapper.js';
+import { APPEAL_CASE_STATUS, APPEAL_VIRUS_CHECK_STATUS } from 'pins-data-model';
+import { submaps as hasSubmaps } from './has.js';
+import { submaps as fpaSubmaps } from './fpa.js';
+import usersService from '#appeals/appeal-users/users-service.js';
+
+/**
+ * @typedef {import('#appeals/appeal-details/appeal-details.types.js').WebAppeal} WebAppeal
+ * @typedef {import('../../../app/auth/auth-session.service.js').SessionWithAuth} SessionWithAuth
+ */
+
+/**
+ * @typedef {(
+ *   appealDetails: WebAppeal,
+ *   currentRoute: string,
+ *   session: SessionWithAuth,
+ *   skipAssignedUsersData?: boolean
+ * ) => Promise<{appeal: MappedInstructions}>} Mapper
+ */
+
+/**
+ * @typedef {Object} SubMapperParams
+ * @property {WebAppeal} appealDetails
+ * @property {string} currentRoute
+ * @property {SessionWithAuth} session
+ * @property {boolean} [skipAssignedUsersData]
+ * @property {{
+ *     [x: string]: any;
+ *   } | null} [caseOfficerUser]
+ * @property {{
+ *     [x: string]: any;
+ *   } | null} [inspectorUser]
+ */
+
+/**
+ * @typedef {(params: SubMapperParams) => Instructions} SubMapper
+ */
+
+// Not super buzzed about these keys, is there something nicer than appealType we can use here?
+/** @type {Record<string, Record<string, SubMapper>>} */
+const submaps = {
+	Householder: hasSubmaps,
```

Comment:
You could probably just use the agreed shorthand appeal type names as keys here - `has` and `fpa` - it would be nice if those were globally defined somewhere but I don't think that's the case yet